src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy

```diff
@@ -180,6 +200,9 @@ public class ArtifactoryPermissionsUpdater {
      */
     private static void submitPermissionTargets(File jsonApiFileDir) {
         jsonApiFileDir.eachFile { file ->
+            if (file.name == 'github-index.json') {
```

Comment:
Would prefer a more magical file name that cannot conceivably clash with a plugin name. `github.index.json` or `_github-index.json` would do it.

Reply:
In fact, it does not seem this file should even be here. AFAIR this directory contains the API payloads for Artifactory, so this should be stored elsewhere.

Reply:
> a more magical file name that cannot conceivably clash with a plugin name

I can try to come up with something, on the off chance someone decides to write a plugin named `github-index`.

> this should be stored elsewhere

That then requires a new input parameter for a new temporary directory. And a patch to `Jenkinsfile`, which as a submitter without write permission to the repository, I cannot then test. Seems like a hassle.

Reply:
> a new input parameter for a new temporary directory

Probably easy enough to do downstream later? Or is your use of this tied to specific file paths?